pkg/ibmvpc/ibmDrawioGenerator.go

```diff
@@ -0,0 +1,79 @@
+package ibmvpc
+
+import (
+	"github.com/np-guard/vpc-network-config-analyzer/pkg/drawio"
+	"github.com/np-guard/vpc-network-config-analyzer/pkg/vpcmodel"
+)
+
+// implementations of the GenerateDrawioTreeNode() for resource defined in ibmvpc:
+func (v *VPC) GenerateDrawioTreeNode(gen *vpcmodel.DrawioGenerator) drawio.TreeNodeInterface {
+	return drawio.NewVpcTreeNode(gen.Cloud(), v.Name())
+}
+
+func (z *Zone) GenerateDrawioTreeNode(gen *vpcmodel.DrawioGenerator) drawio.TreeNodeInterface {
+	return drawio.NewZoneTreeNode(gen.TreeNode(z.VPC()).(*drawio.VpcTreeNode), z.name)
+}
+
+func (s *Subnet) GenerateDrawioTreeNode(gen *vpcmodel.DrawioGenerator) drawio.TreeNodeInterface {
+	// todo - how to handle this error:
+	zone, _ := s.Zone()
+	zoneTn := gen.TreeNode(zone).(*drawio.ZoneTreeNode)
+	return drawio.NewSubnetTreeNode(zoneTn, s.Name(), s.cidr, "")
+}
+
+func (sgl *SecurityGroupLayer) GenerateDrawioTreeNode(gen *vpcmodel.DrawioGenerator) drawio.TreeNodeInterface {
+	tn := drawio.NewSGTreeNode(gen.TreeNode(sgl.VPC()).(*drawio.VpcTreeNode), sgl.Name())
+	for _, sg := range sgl.sgList {
+		for _, ni := range sg.members {
+			gen.TreeNode(ni).(*drawio.NITreeNode).SetSG(tn)
+		}
+	}
+	return tn
+}
+
+func (nl *NaclLayer) GenerateDrawioTreeNode(gen *vpcmodel.DrawioGenerator) drawio.TreeNodeInterface {
+	for _, acl := range nl.naclList {
+		for _, sn := range acl.subnets {
+			gen.TreeNode(sn).(*drawio.SubnetTreeNode).SetACL(acl.Name())
+		}
+	}
+	return nil
+}
+
+func (ni *NetworkInterface) GenerateDrawioTreeNode(gen *vpcmodel.DrawioGenerator) drawio.TreeNodeInterface {
+	return drawio.NewNITreeNode(
+		gen.TreeNode(ni.subnet).(drawio.SquareTreeNodeInterface),
+		nil, ni.Name())
+}
+func (n *IKSNode) GenerateDrawioTreeNode(gen *vpcmodel.DrawioGenerator) drawio.TreeNodeInterface {
+	return nil
```

Comment:
Is this planned to be supported? add some comment?

Reply:
actually, I do not know what it is. 
adding comment